lib/recaptcha/configuration.rb

```diff
@@ -46,12 +46,12 @@ def initialize # :nodoc:
       @skip_verify_env = %w[test cucumber]
       @handle_timeouts_gracefully = true
 
-      @secret_key = ENV['RECAPTCHA_SECRET_KEY']
-      @site_key = ENV['RECAPTCHA_SITE_KEY']
+      @secret_key = ENV['RECAPTCHA_SECRET_KEY'] # rubocop:disable Style/FetchEnvVar
+      @site_key = ENV['RECAPTCHA_SITE_KEY'] # rubocop:disable Style/FetchEnvVar
```

Comment:
best disable this whole rule it's kinda annoying 🤷

Reply:
Done and rebased.